WidgetLibrary/WidgetLibrary/Default.aspx.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace WidgetLibrary
+{
+    public partial class Default : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+
+        }
```

Comment:
Not a big deal at this point in time, but in real life, I'd remove any unused event handlers.